recipes/angelscript/all/conandata.yml

```
@@ -1,7 +1,24 @@
 sources:
+  "2.36.0":
+    url: "https://www.angelcode.com/angelscript/sdk/files/angelscript_2.36.0.zip"
+    sha256: "33f95f7597bc0d88b097d35e7b1320d15419ffc5779851d9d2a6cccec57811b3"
+  "2.35.1":
+    url: "https://www.angelcode.com/angelscript/sdk/files/angelscript_2.35.1.zip"
+    sha256: "5c1096b6d6cf50c7e77ae93c736d35b69b07b1e5047161c7816bca25b413a18b"
   "2.35.0":
-    url: "http://www.angelcode.com/angelscript/sdk/files/angelscript_2.35.0.zip"
+    url: "https://www.angelcode.com/angelscript/sdk/files/angelscript_2.35.0.zip"
     sha256: "010dd45e23e734d46f5891d70e268607a12cb9ab12503dda42f842d9db7e8857"
+patches:
+  "2.36.0":
+    - patch_file: "patches/2.36.0-0001-missing-include-stdint.patch"
+      patch_description: "Add missing stdint.h include"
```

Maybe report it upstream 🙏 

```suggestion
      patch_description: "Add missing stdint.h include"
      patch_type: "portability"
```